### DISCUSSION TITLE
RaspberryMatic auf Intel BNUC11ATKC2 scheitert

### DISCUSSION THREAD

**Discussion by FFellheim** [OpenCCU/OpenCCU#2343] [🇩🇪 Fragen&Antworten] [answered]
Hallo Community,
als Neuling habe ich ein Problem mit der Installation von RaspberryMatic auf einem Intel NUC11 (BNUC11ATKC2). Es mag mir nicht gelingen, den NUC vom USB-Stick zu booten und RaspberryMatic danach zu installieren. 

Der Stick wurde exakt nach dem Leitfaden von Jens Maus erstelltt. Dennoch findet der NUC kein Bootmedium, unabhängig davon, welche Booteinstellungen ich auch gewählt habe.

Ein technischer Fehler des NUC oder des USB-Sticks scheidet aus, denn eine Installation von HomeAssistant klappt auf Anhieb und problemlos.

Kann mir jemand einen Hinweis geben, worin mein Scheitern seine Ursache haben kann?

Besten Dank und viele Grüße
Florian

**Comment by jens-maus** [comment, marked as answer]
Das könnte daran liegen, das eine NUC11 und neuer kein Legacy-Boot mehr unterstützt und nur auf UEFI Boot setzt welches aber vom RaspberryMatic Image leider nicht unterstützt wird.

**Reply by FFellheim** [reply]
Vielen Dank für die schnelle Rückmeldung. Ich hatte dies bereits für möglich gehalten.
Schade, dass aktuelle NUCs dadurch für die RaspberryMatic ausscheiden.
Ich werde wohl versuchen, eine NUC10 zu finden.
Beste Grüße
Florian

**Reply by jens-maus** [reply]
Ich würde eher dazu raten statt RaspberryMatic direkt auf der NUC zu installieren stattdessen dort ProxmoxVE als Virtualisierungsplatform zu installieren und dann einfach RaspberryMatic als virtuelle Maschine (OVA) zu installieren und zu nutzen. Eine NUC11 (aber auch eine NUC10) ist IMHO etwas überdimensioniert nur für einen RaspberryMatic Betrieb.

**Reply by FFellheim** [reply]
Vielen Dank für den guten Tipp.
Beste Grüße
Florian

**Reply by FFellheim** [reply]
Habe RaspberyMatic auf ProxmoxVE installiert: hat Dank der Anleitung wunderbar geklappt. Ich werde jetzt experimentieren.
Vielen Dank nochmals für die Tipps :-)

**Comment by Baxxy13** [comment]
Vielleicht lässt sich "Legacy Boot" ja noch aktivieren.
[Legacy Boot kann im Intel® NUC-BIOS nicht aktiviert werden](https://www.intel.de/content/www/de/de/support/articles/000032529/intel-nuc.html)

**Reply by FFellheim** [reply]
Danke für den Hinweis. Ich kannte ihn schon - funktioniert bei meiner NUC aber leider nicht.
